crates/bevy_feathers/src/assets/fonts/FiraSans-License.txt

```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2012-2015, The Mozilla Foundation and Telefonica S.A.
```

Comment:
I think its worth considering [Firava](https://github.com/hellogreg/firava) (a variable font variant of Fira Sans), rather than checking in each font variant. Provided it works, it would give users maximum flexibility and (ideally) cut down on a bit on complexity.
I believe we can render variable fonts, however it looks like we aren't currently wiring up "font width", so idk how functional it would be in the short term. We don't _need_ to resolve this now. Although committing to that path now would mean we aren't adding these fonts to the history, which would be nice.